### ISSUE TITLE
can you support num key in json5

### ISSUE THREAD

**Issue by wanniwa** [dpranke/pyjson5#80]
```json
{
    "Changes": [
        { //HARDER BUILDING CONSTRUCTION		
            "Action": "EditData",
            "Target": "Data/Blueprints",
            "Fields": {
                "Silo": {
                    0: "390 300 330 15 334 5",
                    17: "1000"
                },
                "Mill": {
                    0: "388 250 390 100 428 8",
                    17: "8000"
                },
                "Earth Obelisk": {
                    0: "337 20 86 10",
                    17: "1000000"
                },
                "Water Obelisk": {
                    0: "337 10 372 10 393 10",
                    17: "1000000"
                },
                "Desert Obelisk": {
                    0: "337 25 88 10 90 10",
                    17: "1000000"
                },
                "Junimo Hut": {
                    0: "390 250 268 20 771 100",
                    17: "40000"
                },
                "Gold Clock": {
                    17: "10000000"
                },
                "Stable": {
                    0: "709 120 335 10",
                    17: "30000"
                },
                "Well": {
                    0: "390 150",
                    17: "1000"
                },
                "Shipping Bin": {
                    0: "388 150",
                    17: "750"
                },
                "Coop": {
                    0: "388 400 390 100",
                    17: "5000"
                },
                "Big Coop": {
                    0: "388 550 390 150",
                    17: "12000"
                },
                "Deluxe Coop": {
                    0: "388 700 390 200",
                    17: "25000"
                },
                "Barn": {
                    0: "388 450 390 150",
                    17: "8000"
                },
                "Big Barn": {
                    0: "388 750 390 300",
                    17: "16000"
                },
                "Deluxe Barn": {
                    0: "390 200 330 15 334 5",
                    17: "35000"
                },
                "Slime Hutch": {
                    0: "390 950 338 10 337 3",
                    17: "30000"
                },
                "Shed": {
                    0: "388 300",
                    17: "15000"
                },
                "Big Shed": {
                    0: "388 750 390 300",
                    17: "40000"
                },
                "Fish Pond": {
                    0: "390 250 152 10 153 10",
                    17: "15000"
                },
            },
            "When": {
                "HarderBuildingConstruction": true
            }
        },
    ]
}
```
This is a game mod’s json thas i want to parse .

**Comment by dpranke** [comment]
Unfortunately, that's not legal JSON5 (or JSON). There's not a good way that I can think of to try and overload the parsing behavior to make that work.

**Comment by dpranke** [comment]
Having thought about this some more, I am sympathetic to that being a reasonable file format, even if it's not valid JSON5.

Luckily, perhaps, I realized it is valid [HJSON](https://hjson.github.io/). I tested your string using the Python [hjson](https://pypi.org/project/hjson/), library, and it looked like it loaded fine.

Honestly, if I was picking a data format these days for something that basically had the JSON data model, I'd probably pick HJSON instead of JSON5.
